api/test/integration/test_vulnerability_endpoints.tavern.yaml

```diff
@@ -333,3 +333,63 @@ stages:
           failed_items: []
           total_affected_items: 1
           total_failed_items: 0
+
+
+---
```

Comment:
I would add another test for limit parameter. Its behavior is a little bit different for this endpoint and it should be specifically tested.

Reply:
The limit parameter is being tested in the unit tests as we are using testing databases with static data. Since the vulnerability inventory could change depending on the agent, these integration tests would have to use custom validation functions to test this and I do not think it is worth. Nevertheless, it is up to you.